src/parallel/MPI_TIMED/mpi_timed.h

```diff
@@ -0,0 +1,231 @@
+
+#ifndef MARDYN_MPI_TIMED_H
+#define MARDYN_MPI_TIMED_H
+
+#include <mpi.h>
+#include "parallel/MPI_TIMED/ProcessTimer.h"
+
+extern ProcessTimer _processTimer;
+
+#ifdef __cplusplus
```

Comment:
you can remove this ifdef.
compiling this file with a c compiler will not work because you include parallel/MPI_TIMED/ProcessTimer.h